frontend/src/upload.js

```diff
@@ -141,27 +221,80 @@ $(document).ready(function() {
       }
     });
 
+    let uploadStart;
     fileSender.on('encrypting', isStillEncrypting => {
       // The file is being encrypted
       if (isStillEncrypting) {
         console.log('Encrypting');
       } else {
         console.log('Finished encrypting');
+        uploadStart = new Date().getTime();
       }
     });
-    let t = '';
+
+    let t;
+    const startTime = new Date().getTime();
+
+    let unexpiredFiles = 1;
+
+    for (let i = 0; i < localStorage.length; i++) {
+      const id = localStorage.key(i);
+      if (isFile(id)) {
+          unexpiredFiles += 1;
+      }
+    }
+
+    let totalDownloads = 0;
+    if (localStorage.hasOwnProperty('totalDownloads')) {
+      totalDownloads = localStorage.getItem('totalDownloads');
```

Comment:
we should be consistent with the type of `totalDownloads` and others. localStorage stores everything as a string so helper functions to convert for us will mean less repetition:

```js
function getTotalDownloads() {
  return Number(localStorage.getItem('totalDownloads'))
}
```